src/main/java/uk/gov/pay/webhooks/validations/WebhookRequestValidator.java

```diff
@@ -39,7 +40,11 @@ public void validate(JsonNode payload, Boolean isLiveContext)  {
     //                callback url. This will require guice injection in the validator contexts and is out of scoped here
     //                but would allow for all of the validation to be processed in one place
     public void validate(CreateWebhookRequest createWebhookRequest) {
+        MDC.put("gateway_account_id", createWebhookRequest.gatewayAccountId());
+        MDC.put("service_id", createWebhookRequest.serviceId());
+        MDC.put("live", createWebhookRequest.live().toString());
         callbackUrlService.validateCallbackUrl(createWebhookRequest.callbackUrl(), createWebhookRequest.live());
+        MDC.clear();
```

Comment:
will this be called if the above line throws an error? Better to have MDC cleanup in a `finally` block

Reply:
good point